backend/app/services/anti_hallucination_service.py

```
@@ -0,0 +1,155 @@
+            "metric", "具体数字指标需要数据来源支撑",
+            default_severity="warning",
+        ))
+
+        return issues
+
+    def scan_chapters(
+        self,
+        chapters: list[dict],
+        evidence_refs: list[dict] | None = None,
+    ) -> dict[str, list[HallucinationIssue]]:
+        """批量扫描多个章节，返回 {chapter_id: [issues]}"""
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)